### PR TITLE
kodi: cleanup redundant AML aarch64 specific sysinfo

### PR DESCRIPTION
**Pull request by wrxtasy**
The Kodi sysinfo screen is showing either N/A or info that is not relevant for AML aarch64 and the common linux-amlogic Kernel we use.

Remove redundant info and tidy up.

### CODE REVIEW

File: packages/mediacenter/kodi/patches/aarch64/kodi-0007-improve-sysinfo-for-AML-aarch64.patch [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+     SetControlLabel(i++, "%s: %s", 12394, SYSTEM_TOTALUPTIME);
+-    SetControlLabel(i++, "%s: %s", 12395, SYSTEM_BATTERY_LEVEL);
+   }
+ 
+   else if (m_section == CONTROL_BT_STORAGE)
+@@ -132,8 +131,6 @@ void CGUIWindowSystemInfo::FrameMove()
+     SetControlLabel(i++, "%s: %s", 150, NETWORK_IP_ADDRESS);
+     SetControlLabel(i++, "%s: %s", 13159, NETWORK_SUBNET_MASK);
+     SetControlLabel(i++, "%s: %s", 13160, NETWORK_GATEWAY_ADDRESS);
+-    SetControlLabel(i++, "%s: %s", 13161, NETWORK_DNS1_ADDRESS);
```
Comment (chewitt) [inline packages/mediacenter/kodi/patches/aarch64/kodi-0007-improve-sysinfo-for-AML-aarch64.patch]: No. DNS servers must be left in.

Comment (wrxtasy) [inline packages/mediacenter/kodi/patches/aarch64/kodi-0007-improve-sysinfo-for-AML-aarch64.patch]: Is this for VPN's ?
As I've never seen DNS servers ever on shown aarch64 platforms.

**Comment by lrusak** [comment]
I'm not really interested in patching Kodi just to display or not display some items.

**Comment by chewitt** [comment]
@wrxtasy I'm closing these PR's down. I fully agree Kodi is showing some unimportant info but refinements to the system-info display should be done upstream from a perspective of all Kodi build projects instead of adding device or arch specific patches. Perhaps have a think about how to hide elements with null values, as that would eliminate many things.

**Pull request closed without merging**